cmd/swagger/commands/diff.go

```diff
@@ -39,33 +39,51 @@ func (c *DiffCommand) Execute(args []string) error {
 	log.Printf("IgnoreFile (-i) :%s", c.IgnoreFile)
 	log.Printf("Diff Report Destination (-d) :%s", c.Destination)
 
+	log.Println("Calc diffs")
 	diffs, err := getDiffs(args[0], args[1])
 	if err != nil {
 		return err
 	}
 
-	ignores, err := readIgnores(c.IgnoreFile)
-	if err != nil {
-		return err
+	if len(c.IgnoreFile) > 0 {
+		log.Println("Read ignores")
```

Comment:
is it debug code or is there a reason for this extra verbosity? We already have a decent recap of what is going to happen when the process starts.